testing/sdk_auth.py

```diff
@@ -38,24 +38,27 @@
 # resiliency towards possible intermittent network failures.
 
 
-@retrying.retry(stop_max_attempt_number=3, wait_fixed=2000)
 def _get_kdc_task(task_name: str) -> dict:
     """
     :return (dict): The task object of the KDC app with desired properties to be retrieved by other methods.
     """
-    log.info("Getting KDC task")
-    _, raw_tasks, _ = sdk_cmd.run_cli("task --json", print_output=False)
-    if raw_tasks:
-        tasks = json.loads(raw_tasks)
-        for task in tasks:
-            if task["name"] == task_name:
-                return task
-
-    raise RuntimeError(
-        "Expecting marathon KDC task but no such task found. Running tasks: {tasks}".format(
-            tasks=raw_tasks
+    @retrying.retry(stop_max_attempt_number=3, wait_fixed=2000)
+    def _get_kdc_task(task_name: str) -> dict:
```

Comment:
Can we change this name into something like `_get_kdc_task_retried` to avoid name collision?